pkg/cli/error.go

```diff
@@ -268,6 +268,14 @@ func MaybeDecorateGRPCError(
 			if pgcode.MakeCode(string(wErr.Code)) == pgcode.ProtocolViolation {
 				return connSecurityHint()
 			}
+
+			// Are we running a v20.2 binary against a v20.1 server?
+			if strings.Contains(wErr.Message, "column \"membership\" does not exist") {
+				// The v20.2 binary makes use of columns not present in v20.1,
+				// so this is a disallowed operation. Surface a better error
+				// code here.
+				return fmt.Errorf("cannot use a v20.2 cli against servers running v20.1")
```

Comment:
cc @knz just to make sure this is ok - we want the 20.2 binary to use this new column name (the old one stays around for 20.1 usage). Or is there a simple version check we can use to determine the column to use? EIther way, if you feel like we don't need to support it, I'd prefer to keep this simple.